doc/make.bat

```diff
@@ -11,7 +11,7 @@ if "%SPHINXBUILD%" == "" (
 )
 set SOURCEDIR=source
 set BUILDDIR=_build
-set SPHINXOPTS=-j auto -W --keep-going -w build_errors.txt -N -q
+set SPHINXOPTS=-j auto -W --keep-going -w build_errors.txt -N -v
```

Comment:
I think we were using `-q` here before to suppress some of the build output coming from the Fluent RPC calls which clutter things up when you are trying to find problems with the documentation (eg: warnings etc..)

Reply:
Yes, that's right. but as mentioned above to see the progress and all warnings we have changed it to `-v`. The warnings are helpful to resolve the documentation issues.

We will change this to `-q` again.